### PR TITLE
updates to get_us_regional_cases

### PR DESCRIPTION
**Pull request by hamishgibbs**
CSSEGISandData https://github.com/CSSEGISandData/COVID-19 has changed data formats and no longer returns US state level cases. 

Previous data source has been replaced with New York Times: https://github.com/nytimes/covid-19-data.

Added functionality: 
1. Return county or state level case and death counts.  
2. Return summary of total cases and deaths or full timeseries.

### CODE REVIEW

File: R/get_us_regional_cases.R [hunk 1]
```diff
@@ -1,57 +1,96 @@
...
+  get_nyt_county <- function(path, out){
+    
+    cases <- mem_read(path)
+    
+    if (out == 'total'){
+      cases <- cases %>%
+        tidyr::drop_na() %>% 
+        dplyr::select(-date) %>% 
+        dplyr::group_by(fips) %>%
+        dplyr::summarize(county = unique(county),
```
Comment (seabbs) [inline R/get_us_regional_cases.R:72]: Dude! z -> s

File: R/get_us_regional_cases.R [hunk 2]
```diff
@@ -1,57 +1,96 @@
...
+                  deaths = sum(deaths, na.rm = TRUE)) %>% 
+        dplyr::arrange(-cases)
+    }
+    
+    
+    return(cases)
+    
+  }
+  
+  if (level == 'state'){
```
Comment (seabbs) [inline R/get_us_regional_cases.R:84]: just a style comment. I must prefer " over ' - why I don't know.....

**Review by seabbs — CHANGES_REQUESTED** [review]
Looks good. Some extremely minor changes suggested. Don't forget to add yourself to the author list (as an author not ctb if not already)

**Review by hamishgibbs — COMMENTED** [review]
updated get_ecdc_cases to use the cache

**Comment by seabbs** [comment]
Looks okay but not passing travis?

**Comment by seabbs** [comment]
This is the issue I am seeing: 

```
odoc mismatches from documentation object 'get_us_regional_cases':
1226get_us_regional_cases
1227  Code: function(level = "state", out = "timeseries")
1228  Docs: function()
1229  Argument names in code not in docs:
1230    level out
```

**Comment by hamishgibbs** [comment]
Docs needed to be updated, I resolved a number of conflicts manually.

**Review by seabbs — APPROVED** [review]
Looks good. Congrats on your first pull request!!

**Pull request merged**